src/olympia/activity/utils.py

```diff
@@ -195,3 +218,20 @@ def filter_queryset_to_pending_replies(queryset, log_type_ids=NOT_PENDING_IDS):
     if not latest_reply:
         return queryset
     return queryset.filter(created__gt=latest_reply.created)
+
+
+def bounce_mail(message, reason):
+    if (not type(message) is dict or
+            ('From' not in message and 'ReplyTo' not in message)):
+        log.error('We needed a From or ReplyTo to bounce but didn\'t get one')
```

Comment:
At this point, we don't have any id or so that we could log right?

I find the message unclear though, I'd rather see something like "Trying to bounce activity mail but no From or ReplyTo header present."